.github/workflows/deploy_gcp_litellm_proxy.yaml

```diff
@@ -59,11 +59,21 @@ jobs:
           zone: "${{ secrets.DEPLOYMENT_ZONE }}"
           ssh_private_key: "${{ secrets.GCP_SSH_PRIVATE_KEY }}"
           command: |
+            gcloud secrets versions access latest \
+              --secret=${{ env.RESOURCE_PREFIX }}-gcp-credential-json \
+              --format='get(payload.data)' \
+              | tr '_-' '/+' \
+              | base64 -d \
+              > /app/credentials.json
```

Comment:
We need the file in the container, so I'm directly downloading the credentials from within the container